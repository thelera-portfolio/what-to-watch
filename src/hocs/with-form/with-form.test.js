import * as React from "react";
import renderer from "react-test-renderer";
import withForm from "./with-form";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

const MockComponentWrapped = withForm(MockComponent);

it(`withForm is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      isDisabled={true}
      isValid={true}
      score={5}
      text={`text`}
      onDisable={() => null}
      onRatingChange={() => null}
      onTextChange={() => null}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
