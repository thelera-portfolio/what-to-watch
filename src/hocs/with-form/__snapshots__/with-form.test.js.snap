// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withForm is rendered correctly 1`] = `<div />`;
